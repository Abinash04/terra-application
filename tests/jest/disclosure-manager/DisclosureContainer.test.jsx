import React from 'react';
import DisclosureContainer from '../../../src/disclosure-manager/_DisclosureContainer';

describe('DisclosureContainer', () => {
  // Snapshot Tests
  it('should render the provided children', () => {
    const disclosureContainer = (
      <DisclosureContainer>
        <div>Test Child</div>
      </DisclosureContainer>
    );

    const wrapper = shallow(disclosureContainer);
    expect(wrapper).toMatchSnapshot();
  });
});
