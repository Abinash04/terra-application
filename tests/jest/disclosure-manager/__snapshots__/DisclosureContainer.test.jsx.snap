// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DisclosureContainer should render the provided children 1`] = `
<ApplicationErrorBoundary>
  <ApplicationLoadingOverlayProvider>
    <withPromptRegistration(NavigationPromptCheckpoint)>
      <BannerCheckpoint>
        <div>
          Test Child
        </div>
      </BannerCheckpoint>
    </withPromptRegistration(NavigationPromptCheckpoint)>
  </ApplicationLoadingOverlayProvider>
</ApplicationErrorBoundary>
`;
